### PR TITLE
chore: update openai compatibility example

### PR DESCRIPTION
**Pull request by Wendong-Fan**
## Description

Describe your changes in detail.

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/camel-ai/camel/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)
- [ ] Example (update in the folder of example)

## Implemented Tasks

- [ ] Subtask 1
- [ ] Subtask 2
- [ ] Subtask 3

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [CONTRIBUTION](https://github.com/camel-ai/camel/blob/master/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by lightaime** [comment]
Hi @Wendong-Fan we can also keep the Qwen example. There is not harm to keep more.

**Comment by lightaime** [comment]
Can we also add a link about how to get the API key?

File: examples/models/openai_compatibility_model_example.py
Comment (lightaime) [inline examples/models/openai_compatibility_model_example.py:1]: Let's create a folder named `openai_compatibility_model_examples`

Comment (Wendong-Fan) [reply examples/models/openai_compatibility_model_example.py:1]: it's a better idea to add examples under the folder

**Pull request merged**